games-engines/chocolate-doom/chocolate_doom-2.2.1.recipe

```diff
@@ -0,0 +1,53 @@
+SUMMARY="Doom source port that accurately reproduces the experience of Doom"
+DESCRIPTION="Chocolate Doom is a source port of the game Doom, by id Software. The purpose of \
+Chocolate Doom is to be as compatible as possible with the original DOS version of Doom."
+HOMEPAGE="http://www.chocolate-doom.org"
+COPYRIGHT="Chocolate Doom Team"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://www.chocolate-doom.org/downloads/$portVersion/chocolate-doom-$portVersion.tar.gz"
+SOURCE_DIR="chocolate-doom-$portVersion"
+CHECKSUM_SHA256="ad11e2871667c6fa0658abf2dcba0cd9b26fbd651ee8df55adfdc18ad8fd674a"
+
+ARCHITECTURES="!x86_gcc2 ?x86 ?x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	chocolate_doom = $portVersion
+	app:chocolate_doom = $portVersion
+        "
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libSDL$secondaryArchSuffix
+	lib:libsdl_mixer$secondaryArchSuffix
+	lib:libsdl_net$secondaryArchSuffix
+	"
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libsdl$secondaryArchSuffix
+	devel:libsdl_mixer$secondaryArchSuffix
+	devel:libsdl_net$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:awk
+	cmd:gcc$secondaryArchSuffix
+	cmd:install
+	cmd:ld$secondaryArchSuffix
+	cmd:make
+	cmd:pkg_config
+	cmd:python
+	cmd:sed
+	"
+
+BUILD()
+{
+	runConfigure ./configure
+        make $jobArgs
+}
+
+INSTALL()
+{
+        cd src
+	mkdir -p $binDir
+        cp chocolate-doom chocolate-doom-setup chocolate-hexen chocolate-hexen-setup chocolate-heretic chocolate-heretic-setup chocolate-server chocolate-setup chocolate-strife-setup chocolate-strife $binDir/
```

Comment:
only one tab indention. also use line breaks with "\" and indent the new lines with another tab.